src/test/java/org/isf/permissions/TestGroupPermissionManager.java

```diff
@@ -27,20 +27,28 @@
 import java.util.List;
 
 import org.isf.OHCoreTestCase;
+import org.isf.menu.manager.UserGroupManager;
+import org.isf.menu.model.UserGroup;
 import org.isf.permissions.manager.GroupPermissionManager;
 import org.isf.permissions.model.GroupPermission;
+import org.isf.permissions.model.Permission;
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 
 class TestGroupPermissionManager extends OHCoreTestCase {
 
+	@Autowired
+	UserGroupManager userGroupManager;
+
 	@Autowired
 	GroupPermissionManager groupPermissionManager;
 
 	@BeforeEach
 	void setUp() {
 		cleanH2InMemoryDb();
+		executeSQLScript("LoadGroupTable.sql");
 		executeSQLScript("LoadPermissionTables.sql");
```

Comment:
where this script is found?

Reply:
In the same place as `LoadPermissionTables.sql`.